runtime/tutor.txt

```diff
@@ -416,6 +415,338 @@ _________________________________________________________________
+
+--> this SENTENCE SHOULD all be in LOWERCASE!
+
+ Note: Conversely, alt-` changes all selected letters to uppercase.
+
+
+
+
+
+
+
+
+=================================================================
+=                            MACROS                             =
+=================================================================
+
+ Macros are a way to record a set of actions that you want
+ to repeat.
+
+ Press Q to start recording a macro, you should see a popup at
+ the bottom of your screen. Press Q again to stop recording.
+ Press q to repeat your recorded macro.
+
+ 1. Move the cursor to the first line below marked -->.
+    Ensure your cursor is on the > of the arrow.
+ 2. Press Q to start recording.
+ 3. Edit the line to look like the bottom one.
+ 4. Exit insert and press Q again to stop recording.
+ 5. Move to the line below and put your cursor on the > again.
+ 6. Press q to repeat the macro.
+
+ --> ... sentence doesn't have it's first and last ... .
+ --> ... sentence doesn't have it's first and last ... .
+ --> This sentence doesn't have it's first and last word.
+=================================================================
+=                             RECAP                             =
+=================================================================
+
+ * Press y to yank (copy) text and p to paste.
+   * Press space-Y and space-P to yank/paste on the system
+     clipboard.
+
+ * Press ~ to alternate case of selected letters.
+   * Press ` and alt-` to set the case of selected layers to
+     upper and lower respectively.
+
+ * Press Q to record and stop recording a macro.
+   * Press q to repeat the recorded macro.
+
+
+
+
+
+
+
+
+=================================================================
+=                      USING THE JUMPLIST                       =
+=================================================================
+
+ Helix can keep track of "jumps" which are big movements, like
+ jumping to the definition of a function in code. It stores
+ these in what's called the jumplist.
+
+ Press C-s (ctrl-s) to manually save your current position to
+ the jumplist.
+
+ Press C-i and C-o to move forward and backwards in the jumplist
+ respectively.
+
+ 1. Press C-s somewhere.
+ 2. Move far away in the file.
+ 3. Press C-o (just once!) to come back to where you saved.
+
+
+
+
+
+=================================================================
+=                      SEARCHING IN FILE                        =
+=================================================================
+
+ Press / to search forward in file, enter to confirm search.
+ Press n to go to the next search match.
+ Press N to go to the previous search match.
+
+ 1. Press / and type in a common word, like 'banana'.
+ 2. Press enter to confirm the search.
+ 3. press n and N to cycle through the matches.
+
+ Like the select command, searching also uses regex.
+
+ Note: To search backwards, press ? (shift-/).
```

Comment:
We might need to note that `n` and `N`, is still in the same direction unlike vim.